README.md

```diff
@@ -1,21 +1,18 @@
 # conjur-api-python3
 
-Python3-based API SDK for [Conjur OSS](https://www.conjur.org/). The repo
-also includes a self-contained CLI tool (`conjur-cli`) that wraps the API
+Command-line interface and programmatic Python access to the Conjur API.
+
+The repo also includes a self-contained CLI tool (`conjur-cli`) that wraps the API
 in a simple executable script/binary.
 
 [![Test Coverage](https://api.codeclimate.com/v1/badges/d90d69a3942120b36785/test_coverage)](https://codeclimate.com/github/cyberark/conjur-api-python3/test_coverage) [![Maintainability](https://api.codeclimate.com/v1/badges/d90d69a3942120b36785/maintainability)](https://codeclimate.com/github/cyberark/conjur-api-python3/maintainability)
 
 ---
 
-### **Status**: Alpha
-
-#### **Warning: Naming and APIs are still subject to breaking changes!**
+### **Status**: GA
```

Comment:
We should actually use the OSS standard for this section in the README with the GA release - does this project meet the standards for Certified? Is it going to be published in the documentation? Are the CLI AND the client library certified, or just the CLI?
```suggestion
## Certification level

{Certified}
![](https://img.shields.io/badge/Certification%20Level-Certified-6C757D?link=https://github.com/cyberark/community/blob/master/Conjur/conventions/certification-levels.md)

This repo is a **Certified** level project. It's been reviewed by CyberArk to verify that it will securely
work with CyberArk DAP as documented. In addition, CyberArk offers Enterprise-level support for these features. For
more detailed  information on our certification levels, see [our community guidelines](https://github.com/cyberark/community/blob/master/Conjur/conventions/certification-levels.md#community).

```

Reply:
@izgeri would you give the SDK a community level or trusted level badge? 
Because for community the description is "It's a community contributed project that is not reviewed or supported by CyberArk." which is not exactly true